web/src/OverviewTable.tsx

```diff
@@ -532,6 +532,22 @@ const columnNameToInfoTooltip: {
       .
     </>
   ),
+  Widgets: (
+    <>
+      <div>You can add custom buttons here to easily perform actions.</div>
+      <div>
+        See{" "}
+        <a
+          href={linkToTiltDocs(TiltDocsPage.CustomButtons)}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          custom buttons docs
+        </a>
+        .
+      </div>
+    </>
```

Comment:
wdyt about 
```
Buttons can be added to resources to easily perform custom actions. See button docs.
```
? maybe? i don't think it needs a line break or needs to be wrapped in a `div`.

Reply:
Thanks!